src/main/java/tasks/InverseFloatingArray.java

```diff
@@ -1,23 +1,23 @@
 package tasks;
 
-import utils.FileUtils;
-
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 public class InverseFloatingArray {
 
+    private TaskCore taskCore;
+
     private List<String> outputMessages = new ArrayList<>();
 
     private String className = InverseFloatingArray.class.getSimpleName();
 
     public void readInput() throws IOException {
```

Comment:
Create interface : BaseTask

create method resolveTask();
implement the interface and Override them in all classes.